easy_phi/tests/handlers_test.py

```diff
@@ -97,12 +101,77 @@ def test_list_scpi_commands(self):
         self.failIf(response.error)
         response_obj = json.loads(response.body)
 
-        self.assertTrue(isinstance(response_obj, list),
-                        "module_scpi_list expected to return list of commands")
+        self.assertIsInstance(
+            response_obj, list,
+            "module_scpi_list expected to return list of commands")
```

Comment:
Test is failing for me:
<code>
======================================================================
FAIL: test_list_scpi_commands (easy_phi.tests.handlers_test.ListSCPICommandsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/tornado/testing.py", line 120, in __call__
    result = self.orig_method(*args, **kwargs)
  File "easy_phi/tests/handlers_test.py", line 109, in test_list_scpi_commands
    "module_scpi_list returned empty list")
AssertionError: module_scpi_list returned empty list

----------------------------------------------------------------------
</code>